data/language/en-GB.yml

```diff
@@ -116,8 +116,8 @@ strings:
   102: Object in the way
   103: Load Game
   104: Save Game
-  105: Quit Scenario Editor
-  106: Quit Game
```

Comment:
Note to self: these strings appear to be used by more than just the top toolbars. They should not be changed.

I'll change this when I rebase this.